docs/controls/Line.md

```diff
@@ -203,10 +203,16 @@ Each line can have a legendOptions object in [Values](#values) level.
 
 | Property Name | Expected | Default | Description                                           |
 | ------------- | -------- | ------- | ----------------------------------------------------- |
-| showShape     | boolean  | true    | Display Shape in the legend.                          |
 | showLine      | boolean  | false   | Display Line in the legend.                           |
+| showShape     | boolean  | true    | Display Shape in the legend.                          |
 | style         | object   | {}      | Any style that can be applied. Refer [Styles](#style) |
 
+#### Optional
+
+| Property Name | Expected | Default | Description                                           |
+| ------------- | -------- | ------- | ----------------------------------------------------- |
```

Comment:
Do you know what other graphs will have this option? We probably update their documentation as well

Reply:
I see legendOptions is only used by pairedResult and Line graphs. I have updated their documentation [here](https://github.com/NarasimhaShenoi/carbon-graphs/commit/a3c0b7b1e03a373d11caeeaa665926ed3bc3a526)

Reply:
Thanks for updating the documentation. Does it mean it is not required for other graphs?

Reply:
The legend option is available only for the line and paired result. It is not required for other graphs.

Reply:
I believe showElement should be available for all the graph types. Other properties like, showShape, showLine and style only applied to line and paired result graphs. So we can expose only showElement on all graph types.

Reply:
Updated documentation for all graphs [here](https://github.com/NarasimhaShenoi/carbon-graphs/commit/cd4edefe5fb917e528894c96dd60d7a29616f179)

Reply:
+1